Updated pulp docs with explicit instructions (#6)
thoth-station/pulp-operate-first-web: usage.md

## Requesting a Python package index on Operate First Pulp

> Request a repository on the Operate first Pulp Python package index for your package.
> [Create a request at
> operate-first/support](https://github.com/operate-first/support/issues/new/choose)
> using the "*New Python package index request*" template.
> 
> Admin of the Operate first Pulp Python package index, would create the index
> and provide the url for the explicit package.
> 

Reviewer: Minor suggestion: put this as steps to follow:

```suggestion
1. Request a repository on the Operate first Pulp Python package index for your package by
[creating a request at
operate-first/support](https://github.com/operate-first/support/issues/new/choose)
using the "*New Python package index request*" template.

2. An admin of the Operate first Pulp Python package index will create the index
and provide the url for the explicit package in the ticket.

```

Currently, only [AICoE-CI](https://github.com/AICoE/aicoe-ci) can publish
Python packages to Pulp as Pulp does not support RBAC. Thus AICoE-CI acts as an
authority for publishing Python packages to Pulp indexes.
Follow instructions:

- Install [aicoe-ci](https://github.com/apps/aicoe-ci) GitHub app in your GitHub repository.

> - Include `.aicoe-ci.yaml` configuration.yaml, a sample can be found [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml).

Reviewer 2: The page links to the example repo in other places:

```suggestion
- Include `.aicoe-ci.yaml` configuration.yaml, a sample can be found [here](https://github.com/thoth-station/aicoe-ci-pulp-upload-example/blob/master/.aicoe-ci.yaml).
```

> - Edit the below snippet with your explicit **index name** and add it in `.aicoe-ci.yaml` configuration yaml

Reviewer 3: ```suggestion
- Edit the snippet below with your explicit **index name** and add it in `.aicoe-ci.yaml` configuration yaml
```

> - Edit the below snippet with your explicit **package name** and add it in `.aicoe-ci.yaml` configuration yaml

Reviewer 2: An index can host multiple packages (I was surprised with the donkeycar name used):

```suggestion
- Edit the below snippet with your explicit **index name** and add it in `.aicoe-ci.yaml` configuration yaml
```

```yaml
package-upload:
  - instance: pulp

>     pulp-index: <package-name>

Reviewer 2: ```suggestion
    pulp-index: <index-name>
```

    allow-sdist: true
    allow-bdist: true
```

> - Create a tag in GitHub and the app would upload the package to operate first pulp python index.

Reviewer 3: ```suggestion
- Create a tag in GitHub and the app will upload the package to operate first pulp python index.
```
